data-governance/no-code-access.mdx

```diff
@@ -40,11 +40,12 @@ which are exposed to your users.
 
 ## One IP address
 
-If you need to set your firewall to allow our agent to exercise your web app, please use this IP address: `35.192.162.70`
+If you need to set your firewall to allow our agent to exercise your web app, please use this IP address: `35.192.162.70` or
```

Comment:
```suggestion
If you need to set your firewall to allow our agent to access your web app, please allow our IP addresses: `35.192.162.70` and `34.159.153.198`.
```

I would not mention that this is a proxy in europe, this might lead people to not allow both - we want both ;)